### PR TITLE
update passive staking proposal

### PR DESCRIPTION
**Pull request by rob-solana**
#### Problem
 proposal had some conflicting info, names for things, no pictures

 #### Summary of Changes
 groom passive staking proposal
 add art
 add mscgen support to proposals and book makefiles

### CODE REVIEW

File: proposals/src/passive-stake-delegation-and-rewards.md [hunk 1]
```diff
@@ -58,10 +58,27 @@ staker's StakeState accounts.  This is the percentage ceiling of the reward.
 * Account::lamports - The accumulated lamports from the commission.  These do not
 count as stakes.
 
-* `authorized_voter_id` - Only this identity is authorized to submit votes.
+* `authorized_vote_signer` - Only this identity is authorized to submit votes, and
```
Comment (garious) [inline proposals/src/passive-stake-delegation-and-rewards.md]: Can we delete this field and have the vote signer create this account?

File: proposals/src/passive-stake-delegation-and-rewards.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ validator.
 
 In the current design each staker creates their own VoteState, and assigns a
 **delegate** in the VoteState that can submit votes.  Since the validator has to
-actively vote for each stake delegated to it, it can censor stakes by not voting
-for them.
+actively vote for each stake delegated to it, validators can censor stakes by
+not voting to them.
```
Comment (garious) [inline proposals/src/passive-stake-delegation-and-rewards.md]: voting to them?

File: proposals/src/passive-stake-delegation-and-rewards.md [hunk 3]
```diff
@@ -58,10 +58,27 @@ staker's StakeState accounts.  This is the percentage ceiling of the reward.
...
+### VoteInstruction::AuthorizeVoteSigner(Pubkey)
+
+* `account[0]` - RW - The VoteState
+  `VoteState::authorized_vote_signer` is set to to `Pubkey`, instruction must by
+   signed by Pubkey
+
+
+### StakeState
+
+A StakeState takes one of two forms, StakeState::Delegate and StakeState::MiningPool.
```
Comment (garious) [inline proposals/src/passive-stake-delegation-and-rewards.md]: Why name the inner thing `Delegate`?

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 2 above)
Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: "to" because a validator has to issue a transaction to every instance of the VoteProgram that delegates to the validator

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 3 above)
Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: because it captures the status of delegation and rewards collected via the node the stake is delegated to

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 1 above)
Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: if we do so, we'll need a field for the validator node's pubkey (who can submit votes?).  using this approach allows us to bypass needing a vote_signer by default

Comment (garious) [reply proposals/src/passive-stake-delegation-and-rewards.md]: I'm confused. Why does the validator node submit votes instead of the vote signer?  I think if there's any default, it should be to use the Vote account's pubkey as the validator's pubkey.

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 3 above)
Comment (garious) [reply proposals/src/passive-stake-delegation-and-rewards.md]: got it, thanks

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 2 above)
Comment (garious) [reply proposals/src/passive-stake-delegation-and-rewards.md]: for them?

File: proposals/src/passive-stake-delegation-and-rewards.md (hunk 1 above)
Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: the validator node ships the vote over gossip.  the (optional) vote signer merely presents KeypairUtil to the node

Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: https://github.com/rob-solana/solana/blob/24dca44e48134a1cc19ff306107224a51e0e228d/consensus.svg

Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: here's the hack for the race: https://github.com/solana-labs/solana/blob/8d032aba9d978df32215cf35867c9afac45c3bca/core/src/staking_utils.rs#L71

Comment (garious) [reply proposals/src/passive-stake-delegation-and-rewards.md]: @rob-solana, It's only the use of `VotingKeypair` (`RemoteVoteSigner`) that's optional.  There will always be a Vote account. The keypair for that Vote account is what you'd put into `RemoteVoteSigner`.  On validator startup,  it registers its own pubkey with RemoteVoteSigner, but that's just to get it over to RemoteVoteSigner. The RemoteVoteSigner then re-signed the transaction with its keypair, which is the owner of the Vote account.

tldr; a big appeal of this design proposal is that `authorized_vote_signer` is no longer needed.

Comment (garious) [reply proposals/src/passive-stake-delegation-and-rewards.md]: Regarding your hack, why not add `config-staking-account` options to `create-staking-account` in solana-wallet (and then send a single transaction)?

Comment (rob-solana) [reply proposals/src/passive-stake-delegation-and-rewards.md]: we tried that earlier...  the issue was when we needed which keypairs

the decision to pursue that hack for Beacons was expediency, and knowing that this proposal was out there, waiting
